btk/draw_blends.py

```diff
@@ -415,8 +415,10 @@ def render_single(self, entry, filt, psf, survey):
         Returns:
             gal (galsim.InterpolatedImage): The galsim profile of a single galaxy
         """
-        k = int(np.random.rand(1) * len(self.catalog))  # catalog_line["btk_index"][0]
-        gal = self.catalog.makeGalaxy(k, gal_type="real", noise_pad_size=0).withFlux(1)
+
+        galsim_catalog = self.catalog.get_galsim_catalog()
+        gal_flux = get_flux(entry['ref_mag'], filt, survey)
+        gal = galsim_catalog.makeGalaxy(entry['btk_index'], gal_type="real", noise_pad_size=0).withFlux(gal_flux)
```

Comment:
Is it important to set the `noise_pad_size` to something ? Yes, maybe looking at some images might be helpful as @thuiop suggested above. Feel free to paste a couple in the PR.

Reply:
I don't know I guess we don't want noise in galaxy images as we add the Poisson noise later in the simulation ?